doc/docs/README.md

```diff
@@ -0,0 +1,31 @@
+# Overview
+**Welcome to TDM!** This documentation should help you get started being productive with TDM.
```

Comment:
I think the Overview should start with a small intro of what a user might able to do using the tool. Kind of a quick one or two lines of why I would want to use it. I shouldn't have to read a lot to understand what the tool is used for.

Reply:
Does the new layout help? Added a little quip, table to illustrate, and removed link headers.

Reply:
Resolved offline.